packages/web-components/fast-foundation/src/tree-item/tree-item.spec.ts

```diff
@@ -179,19 +179,6 @@ describe("TreeItem", () => {
         await disconnect();
     });
 
-    it("should NOT set a tabindex when disabled is `true`", async () => {
-        const { element, connect, disconnect } = await setup();
-
-        element.disabled = true;
-
-        await connect();
-
-        expect(element.hasAttribute("tabindex")).to.equal(false);
-        expect(element.getAttribute("tabindex")).to.equal(null);
-
-        await disconnect();
-    });
-
```

Comment:
@eljefe223 I think we'll want to have comparable tests in Tree View to ensure we these are still working. While I get that the test may not be 1:1, ideally we have checks to ensure feature parity is working.

Reply:
Right, i think we can add a test in tree view to ensure the tree item sets selected, but this specific test is no longer relevant since `disabled` items are now focusable.

Reply:
Thanks for the reminder (insert faceplam emoji 🙂 )